app/experimenter/features/__init__.py

```diff
@@ -71,15 +72,17 @@ def _load_features(cls):
         features = []
 
         for application in NimbusConstants.APPLICATION_CONFIGS.values():
-            application_json_path = os.path.join(
-                settings.FEATURE_MANIFESTS_PATH, f"{application.slug}.json"
+            application_yaml_path = os.path.join(
+                settings.FEATURE_MANIFESTS_PATH, f"{application.slug}.yaml"
             )
 
-            if os.path.exists(application_json_path):
-                with open(application_json_path) as application_json_file:
-                    application_json_data = json.loads(application_json_file.read())
-                    for feature_slug in application_json_data.keys():
-                        feature_data = application_json_data[feature_slug]
+            if os.path.exists(application_yaml_path):
+                with open(application_yaml_path) as application_yaml_file:
+                    application_data = yaml.load(
+                        application_yaml_file.read(), Loader=yaml.Loader
+                    )
+                    for feature_slug in application_data.keys():
+                        feature_data = application_data[feature_slug]
```

Comment:
Oh fantastic I was hopiong it would be this easy 🙏 🙏 🙏 🙏 🙏

Reply:
Yeah, I was expecting more of a mismatch! But the YAML looks structurally identical and things seem to have Just Worked